get_element_cpo/P2769.bs

```diff
@@ -905,11 +897,96 @@ if constexpr (is_reference_v&lt;range_reference_t&lt;Base>>) {
   return static_cast&lt;E>(std::get<ins>`_element`</ins>&lt;N>(*i));
 }
 </pre>
+</blockquote>
+
+## Modify `uses_allocator_construction_args` in <a href="https://eel.is/c++draft/allocator.uses.construction">[**allocator.uses.construction**]</a> ## {#allocator_uses_construction}
+
+<blockquote>
+<pre>
+template&lt;class T, class Alloc, <i>pair-like</i> P>
+  constexpr auto uses_allocator_construction_args(const Alloc& alloc, P&& p) noexcept;
+</pre>
+
+*Constraints*: `remove_cv_t<T>` is a specialization of `pair` and `remove_cvref_t<P>` is not a specialization of
+               `ranges​::​subrange.`
+
+*Effects*: Equivalent to:
+<pre>
+  return uses_allocator_construction_args&lt;T>(alloc, piecewise_construct,
+                                        forward_as_tuple(get<ins>_element</ins>&lt;0>(std::forward&lt;P>(p))),
+                                        forward_as_tuple(get<ins>_element</ins>&lt;1>(std::forward&lt;P>(p))));
+</pre>
+</blockquote>
+
+## Modify `pair` in <a href="https://eel.is/c++draft/pairs.pair">[**pairs.pair**]</a> ## {#pairs_pair}
+
+<blockquote>
+<pre>
+template&lt;class U1, class U2> constexpr explicit(see below) pair(pair&lt;U1, U2>& p);
+template&lt;class U1, class U2> constexpr explicit(see below) pair(const pair&lt;U1, U2>& p);
+template&lt;class U1, class U2> constexpr explicit(see below) pair(pair&lt;U1, U2>&& p);
+template&lt;class U1, class U2> constexpr explicit(see below) pair(const pair&lt;U1, U2>&& p);
+template&lt;<i>pair-like</i> P> constexpr explicit(see below) pair(P&& p);
+</pre>
+
+Let <code><i>FWD</i>(u)</code> be `static_cast<decltype(u)>(u)` <ins>and let <code><i>get-elem</i></code>
+be `get` for the first four overloads and `get_element` for the last overload.</ins>
```

Comment:
Why not use get_element everywhere?

Reply:
Technically, we can, though it's a change for all other constructors. If that's is something that C++ standard committee prefers, I can do that. Though it needs to be proven with the implementation experience